akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/RouteDirectivesSpec.scala

```diff
@@ -93,7 +93,8 @@ class RouteDirectivesSpec extends AnyWordSpec with GenericRoutingSpec {
                 case AlreadyRegistered =>
                   import spray.json.DefaultJsonProtocol._
                   import SprayJsonSupport._
-                  StatusCodes.BadRequest -> Map("error" -> "User already Registered")
+                  implicit val preciseFormat: spray.json.RootJsonFormat[Map[String, String]] = mapFormat // Scala 3 workaround
+                  ToResponseMarshallable(StatusCodes.BadRequest -> Map[String, String]("error" -> "User already Registered"))
```

Comment:
Weird, indeed, don't know what's going on. `sprayJsonMarshaller[Map[String, String]]` works but finding it implicitly seems to take a wrong path somewhere...

```suggestion
                  StatusCodes.BadRequest -> Map[String, String]("error" -> "User already Registered")
```